inference-engine/tests/functional/inference_engine/keep_constant_inputs_tests.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <iostream>
+#include <cpp/ie_cnn_network.h>
+#include <cnn_network_impl.hpp>  // deprecated API
+
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph_ops/convolution_ie.hpp>
+#include <transformations/init_node_info.hpp>
+
+#include <ie_precision.hpp>
+#include <functional_test_utils/precision_utils.hpp>
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+#include "low_precision_transformations/network_helper.hpp"
+#include <convert_function_to_cnn_network.hpp>
+#include <ngraph_ops/fully_connected.hpp>
+#include <transformations/common_optimizations/common_optimizations.hpp>
+#include <transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.hpp>
+#include <transformations/convert_opset2_to_opset1/convert_opset2_to_opset1.hpp>
+#include <transformations/convert_opset3_to_opset2/convert_opset3_to_opset2.hpp>
+#include "generic_ie.hpp"
+#include "graph_tools.hpp"
+#include "functional_test_utils/low_precision_transformations/layer_transformation.hpp"
+
+using namespace testing;
+using namespace InferenceEngine;
+
+int numberOfInputsForLayerInCNNNetwork(const InferenceEngine::CNNNetwork& network, std::string layerType) {
+    int numberOfInputs = 0;
+    InferenceEngine::CNNLayerPtr layer = nullptr;
+
+    IE_SUPPRESS_DEPRECATED_START
+    for (auto it = details::CNNNetworkIterator(network); it != details::CNNNetworkIterator(); it++) {
+        layer = *it;
```

Comment:
nit: layer can be a local variable (defined inside the cycle)

Reply:
Done.